.eslintrc.js

```diff
@@ -2,7 +2,12 @@
 
 module.exports = {
   root: true,
-  extends: ["plugin:vue/recommended", "prettier", "prettier/vue"],
+  extends: [
+    "plugin:vue/recommended",
```

Comment:
in this way we override again the rules, basically this won't change prettier behaviour much

Reply:
I thought the wished outcome is to fix the html in the vue templates. This is now the case.